goagen/gen_app/generator_test.go

```diff
@@ -437,10 +437,11 @@ func MountWidgetController(service goa.Service, ctrl WidgetController) {
 	mux := service.{{if .version}}Version("{{.version}}").ServeMux(){{else}}ServeMux(){{end}}
 	h = func(c *goa.Context) error {
 		ctx, err := NewGetWidgetContext(c)
-{{if .version}}		ctx.APIVersion = service.Version("{{.version}}").VersionName()
-{{end}}		if err != nil {
+		if err != nil {
 			return goa.NewBadRequestError(err)
 		}
+{{if .version}}		ctx.APIVersion = service.Version("{{.version}}").VersionName()
+{{end}}
```

Comment:
We're not very consistent with blank lines preceding an ending `return` statement.  I prefer the extra line, myself.